includes/wikia/VariablesBase.php

```diff
@@ -8866,13 +8866,6 @@
  */
 $wgYoukuConfig['playerColor'] = 0;
 
-/**
- * Used for test wikis copied on prod. Top articles data should use pageviews of the original wiki.
- * @see PLATFORM-3671
- * @var int $wgDataMartOriginalCityId
- */
-$wgDataMartOriginalCityId = 0;
```

Comment:
I think you've missed this line: https://github.com/Wikia/app/blob/dfa23b940bf8c5be30bd05f78efa1a13f5a5d7e5/includes/wikia/api/ArticlesApiController.class.php#L139